staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -84,7 +89,14 @@ type RequestDigest struct {
 // this type and cfgMeal follow the convention that the suffix
 // "Locked" means that the caller must hold the configController lock.
 type configController struct {
-	queueSetFactory fq.QueueSetFactory
+	name                string // varies in tests of fighting controllers
+	ignoreNotifications bool   // for testing
```

Comment:
Having test branches is something both @lavalamp and I have reservations about.

We don't place test only flags like this in the controllers.  We either factor the controller take fake object input, manually force the notifications, or create a private function which can be replaced (the `sync` funcs in pkg/controllers).  The private function is different because the piece substituted doesn't even get compiled in during the build.

Reply:
Revised, no more `ignoreNotifications` in the controller.